src/keria/app/ipexing.py

```diff
@@ -148,7 +148,7 @@ def sendMultisigExn(agent, hab, ked, sigs, atc, rec):
         serder = serdering.SerderKERI(sad=admitked)
         ims = bytearray(serder.raw) + pathed['exn']
         agent.hby.psr.parseOne(ims=ims)
-        agent.exchanges.append(dict(said=serder.said, pre=hab.pre, rec=[grant.ked['i']], topic="credential"))
+        agent.exchanges.append(dict(said=serder.said, pre=hab.pre, rec=[admitked['a']['i']], topic="credential"))
```

Comment:
@2byrds I actually changed this to be more consistent with the rest of the messages. `rec` here determines which service endpoint it will go to, not the recipient (rp) fields of the exn message which is set on the client side.

The effect of relying on `grant.ked['i']` is a client could incorrectly admit it to the wrong prefix, and it would send to the discloser of the grant but the `rp` field wouldn't match. Better to do as the client asked for.

Perhaps we should have some assertions that the admit matches the grant prefix? What do you think?

Reply:
Also note still using `a.i` here instead of `rp` just for consistency. Yet to migrate everything to `rp`

Reply:
Sounds good to me @iFergal